packages/common/container-definitions/api-report/container-definitions.legacy.alpha.api.md

```diff
@@ -201,6 +201,21 @@ export type IContainerPolicies = {
     maxClientLeaveWaitTime?: number;
 };
 
+// @alpha @legacy
+export interface IContainerStorageService extends Partial<IDisposable> {
```

Comment:
should this directly implement IDisposable and deprecate it?

Reply:
Good point. They likely need to be deprecated or I cannot remove them later I believe. Updated it.